src/libcore/any.rs

```diff
@@ -73,19 +73,16 @@ use crate::intrinsics;
 /// Most types implement `Any`. However, any type which contains a non-`'static` reference does not.
 /// See the [module-level documentation][mod] for more details.
 ///
+/// This trait is notably `unsafe`. However, this does not affect any use of
+/// this trait in downstream code, as it is always safe to use `Any`. The reason
+/// that `Any` is unsafe is that the standard library relies on its
```

Comment:
This makes it sound like there'd be something odd here, but this is entirely normal for unsafe traits: they are unsafe to *implement* but safe to *use*.